<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hill</class>
 <widget class="QWidget" name="Hill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифр Хилла</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="MessageEdit"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="AlphabetLabel">
     <property name="text">
      <string>Алфавит</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="CodeLabel">
     <property name="text">
      <string>Криптограмма</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="MessageLabel">
     <property name="text">
      <string>Сообщение</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="CodeEdit"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="DecodeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Расшифровать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="AlphabetEdit">
     <property name="text">
      <string>ABCDEFGHIJKLMNOPQRSTUVWXYZ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="EncodeButton">
     <property name="text">
      <string>Зашифровать</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableWidget" name="KeyTable">
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>15</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QFrame" name="KeyFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="KeyLabel">
        <property name="text">
         <string>Матрица ключа</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="KeySizeSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
